src/App.js

```diff
@@ -2,15 +2,65 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState(prevState => (
+      { count: prevState.count + 1 }
+    ));
+  };
 
-  increase = () => {};
+  add100 = () => {
+    this.setState(prevState => (
+      { count: prevState.count + 100 }
+    ));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
```

Comment:
```suggestion
    if (this.state.count % 5 === 0 && this.state.count !== 0) {
```
Можно добавить еще такую проверку. Технически 0 на 5 делится, но очень логично)

Reply:
Good point